Microsoft.Toolkit.Uwp.UI.Animations/CompositionAnimations/Animations/RotationInDegreesAnimation.cs

```diff
@@ -0,0 +1,30 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.UI.Composition;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// ScalarAnimation that animates the <see cref="Visual.RotationAngleInDegrees"/> property
+    /// </summary>
+    public class RotationInDegreesAnimation : ScalarAnimation
```

Comment:
Could we just instead have a `Unit` dependency property on the RotationAnimation which specifies an enum for `Radians` or `Degrees`?  It'd be nice to default to Degrees over Radians (like we default for the Rotate helper currently in the toolkit) and have a less verbose name/setup.

Reply:
The naming comes from the property name on the visual that is animated. Visual has both Rotation and RotationInDegrees, so I thought it's better to keep it consistent